cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -679,10 +679,14 @@ def _fail_task(self, task, reason):
     def upload(self, req):
         # type: (Request) -> None
 
+        task_uuid = req[http.REQUEST_HEADER].get('X-IMAGE-UUID', '')
         try:
             UploadHandler(req, self.upload_tasks).handle_upload()
         except Exception, e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)